### ISSUE TITLE
Error with HtmlContentEditor

### ISSUE THREAD

**Issue by renattomachado** [volkanceylan/Serenity#2805]
Hello, using HtmlContentEditor, when I open Add Dialog, the ck editor works perfect, but when I open dialog in edit mode, show this error on browser:

Uncaught TypeError: Cannot read property 'getSelection' of undefined
    at CKEDITOR.dom.selection.getNative (<anonymous>:418:125)
    at new CKEDITOR.dom.selection (<anonymous>:416:70)
    at a.CKEDITOR.editor.getSelection (<anonymous>:413:288)
    at a.elementPath (<anonymous>:252:166)
    at $.refresh (<anonymous>:770:199)
    at $.g (<anonymous>:761:487)
    at a.n (<anonymous>:10:224)
    at a.CKEDITOR.event.CKEDITOR.event.fire (<anonymous>:12:47)
    at a.CKEDITOR.editor.CKEDITOR.editor.fire (<anonymous>:13:242)
    at <anonymous>:332:104

.. and in some times this error don't ocurr.

My form class:

```
   [HtmlContentEditor(Rows = 5)]
   public String Biografia { get; set; }
```

@volkanceylan, do you know what's can to be?

**Comment by Zahar661** [comment]
+1
The first item opens well, on the second error.

**Comment by renattomachado** [comment]
@volkanceylan or @VictorTomaili or #anybody?

**Comment by renattomachado** [comment]
@Zahar661 do you solve that?

**Comment by Zahar661** [comment]
in Customers -> Notes  work well. I made the form as in that example and it works. The customary scheme [HtmlContentEditor] does not work ((

**Comment by Zahar661** [comment]
in 3.3.7 WORK!!!

**Comment by volkanceylan** [comment]
Yes, thats fixed in in 3.3.7
